components/Info.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import { MdInfo } from "react-icons/md"; //Icons
+
+export default function Info({ children }) {
+  return (
+    <InfoWrapper>
+      <MdInfo size="24px" />
+      <InfoText>{children}</InfoText>
+    </InfoWrapper>
+  );
+}
+
+const InfoWrapper = styled.div`
+  display: inline-flex;
+  align-items: center;
+  gap: 10px;
+  background: var(--card-bg);
+  padding: 13px 19px 13px 32px;
+  border-radius: 0 13px 13px 0;
+  border: 1px solid var(--error);
+  border-left: none;
+  z-index: 4000;
+  width: calc(100% - 50px);
```

Comment:
why not give the parent of this infoWrapper a padding of 25px? Or set the margin to 25px?

Reply:
@F-Kirchhoff because I position the info component absolutely & want to have distance to the right border (mobile). I cannot solve this with `margin` because this would have no effect if I position it `absolute` & give `width: 100%`.